src/TimePicker.purs

```diff
@@ -241,6 +245,20 @@ embeddedInput state =
   , timeUnits: state.timeUnits
   }
 
+embeddedReceive :: forall m. CompositeInput -> CompositeComponentM m Unit
+embeddedReceive input = case input.interval of
+  Nothing -> pure unit
```

Comment:
If `input.interval` was `Just _` and it changes to `Nothing`, shouldn't we handle that?

Reply:
Yeah you are right. Will fix.

Reply:
Fixed.